brewtils/models.py

```diff
@@ -699,6 +703,7 @@ def status(self):
 
     @status.setter
     def status(self, value):
+        self.status_updated_at = datetime.now()
```

Comment:
This logic really belongs in the beer-garden side of things.  Really beer-garden shouldn't use these models at all, but that's not the situation we're in right now unfortunately, which creates lots of opportunity for this sort of confusion.  In any case, having this like this on the model on the beer-garden side, which is the thing that all of the writes to the database ultimately go through, ensures that this timestamp gets set correctly in pretty much all circumstances.

Also, not too relevant given what I just said, but as written this wouldn't actually set the `status_updated_at` timestamp on a newly instantiated Request.

Reply:
> In any case, having this like this on the model on the beer-garden side, which is the thing that all of the writes to the database ultimately go through, ensures that this timestamp gets set correctly in pretty much all circumstances.

Using the model's setter method for `status` ensures the timestamp gets set *every* time, I'm not sure I understand the reasoning behind not using Brewtils, but I've tried to update the PRs accordingly.